src/compiler/compile/nodes/Element.ts

```diff
@@ -496,6 +506,18 @@ export default class Element extends Node {
 				}
 			}
 		}
+
+		if (a11y_no_onchange.has(this.name)) {
+			const change_handler = handlers_map.get('change');
```

Comment:
nit: why not `handlers_map.has('change')` since we are just concern about the existence of the key.

Reply:
Makes sense, I made the change